src/main/java/co/rsk/bitcoinj/script/ErpFederationRedeemScriptParser.java

```diff
@@ -41,7 +41,7 @@ public static Script extractStandardRedeemScript(List<ScriptChunk> chunks) {
             throw new VerificationException(message);
         }
 
-        return new Script(chunksForRedeem);
+        return (new Script(chunksForRedeem)).getChunks();
```

Comment:
```suggestion
        return chunksForRedeem;
```
That should be it, right? Why build the script from the chunks to then get the chunks from it?